docs/running-deploying.md

```diff
@@ -28,13 +28,26 @@ make install-requirements
 
 ## Running the Simulator Locally
 
-Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./config.md) for details on how to do this.
-
-To run the simulated API, use the following command from the repository root directory:
-
-```console
-make run-simulated-api
-```
+1. Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./config.md) for details on how to do this.
+
+    The minimum set of environment variables you'll need in your `.env` file to run the simulator locally are as follows:
+
+    ```dotenv
+    SIMULATOR_API_KEY=my-test-key
+    TEST_OPENAI_ENDPOINT=http://localhost:8000/
+    TEST_OPENAI_KEY=my-test-key
+    TEST_OPENAI_DEPLOYMENT=gpt-3.5-turbo-0613
+    ```  
+
+2. Start the simulator by running the following command in your terminal from the repository root directory:
+  
+    ```console
+    make run-simulated-api
+    ```
+
+    Kill the process and run the command again to restart the simulator whenever you make changes to the config.
+3. Now open the [test-aoai.http](test-aoai.http) file, and send the first POST request.  
```

Comment:
This link doesn't resolve - based on the env var, I assume this is intended to work with the chat completions http file?

```suggestion
3. Now open the [http/chat-completions.http](../http/chat-completions.http) file, and send the first POST request.
```

Reply:
fixed